src/Testcontainers.Spanner/SpannerContainer.cs

```diff
@@ -0,0 +1,153 @@
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Testcontainers.Spanner.RestApi.v1;
+
+namespace Testcontainers.Spanner;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public sealed class SpannerContainer : DockerContainer, IDisposable
+{
+  internal const int InternalGrpcPort = 9010;
+  internal const int InternalRestPort = 9020;
```

Comment:
Please move the default ports to the builder (like the other modules are configured).

Reply:
ok